service-front/module/Application/view/application/pages/post_office/donor_details_match_check.twig

```diff
@@ -0,0 +1,48 @@
+{% include 'layout/id_check_banner.twig' with details_data %}
+<div class="govuk-width-container govuk-!-padding-top-5">
+
+        <form method="POST">
+            <h1 class="govuk-heading-xl govuk-!-width-two-thirds">Do the details match the ID document?</h1>
+
+            <p class="govuk-body govuk-!-width-two-thirds">
+                The donor’s name and date of birth recorded in Sirius should be identical to
+                how they are presented on the ID document they have chosen to prove their
+                identity with.
+                <br><br>
+                The donor’s address recorded in Sirius should be their home address as this is
+                used for the identity check.
+                <br><br>
+                If there is a mis-match, amend in Sirius as appropriate.
+            </p>
```

Comment:
This is a nit, but more semantic to use multiple `<p>` instead of newlines
```suggestion
            <p class="govuk-body govuk-!-width-two-thirds">
                The donor’s name and date of birth recorded in Sirius should be identical to
                how they are presented on the ID document they have chosen to prove their
                identity with.
            </p>
            <p class="govuk-body">
                The donor’s address recorded in Sirius should be their home address as this is
                used for the identity check.
            </p>
            <p class="govuk-body">
                If there is a mis-match, amend in Sirius as appropriate.
            </p>
```